src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests_WellKnownAttributes.cs

```diff
@@ -5968,6 +5991,10 @@ public class TestClass
     [Obsolete(""Do not use Prop1"", false)]
     public int Prop1 { get; set; }
 
+    public int Prop2 { [Obsolete(""Do not use Prop2.Get"")] get; set; }
```

Comment:
Could you test a scenario with Obsolete on both the property and the accessor? I think that will produce a single diagnostic.

Reply:
What if one is an error, and one a warning?
Or both an error/warning, but with different messages?
It might be best if we treat it as two separate diagnostics.
We could just ban making both a property and an accessor obsolete, but that still wouldn't help when consuming the attribute from other languages.

Reply:
From discussion with @cston two options seem ok:
- report the worse one
- report both

Either seems fine, whatever is easiest to implement.

Reply:
It might be reasonable to give priority to the attribute on the accessor.

Reply:
The current behaviour is to report both. Given that this is both an unlikely scenario, and the correct behaviour is debatable, I think it makes most sense to take the simplest option, and keep the current behaviour.